src/modules/stripe/stripe.controller.ts

```diff
@@ -0,0 +1,35 @@
+import {
+  Controller,
+  HttpCode,
+  Post,
+  Req,
+  Headers,
+  RawBodyRequest,
+  Body,
+  UseGuards,
+} from '@nestjs/common';
+import { StripeService } from './stripe.service';
+import { Request } from 'express';
+import { CreateCheckoutDto } from './dto';
+import { AuthTokenGuard } from '../../shared/guards';
+
+@Controller('stripe')
+export class StripeController {
```

Comment:
Let's name this PaymentService. And we should rename other components like module and service names to not be stuck to the service name (Stripe) as well.